static/app/views/releases/detail/overview/index.tsx

```diff
@@ -659,14 +659,14 @@ function getDropdownOptions(): DropdownOption[] {
     },
     {
       sort: {kind: 'desc', field: 'trend_percentage()'},
-      query: [['t_test()', '<-6']],
```

Comment:
is the change from `<-6` to `>6` intended?

Reply:
:+1: the trends endpoint uses the trendType parameter to determine the condition correctly so the user doesn't have to